### DISCUSSION TITLE
About the explanation of the explorer of Mastodon official document.

### DISCUSSION THREAD

**Discussion by TaikiUozumi** [mastodon/mastodon#25782] [General]
The explanation of public uris and /explore in the official document (https://docs.joinmastodon.org) seems wrong.

URL: https://docs.joinmastodon.org/dev/routes/#public

Public URIs
/explore = profile directory

Shouldn't the above part be like this?
/explore = explore
/directory = profile directory

Official document (other part)
https://docs.joinmastodon.org/user/discoverability/#directory

The profile directory shows all accounts that have opted into being shown in the directory, and can be used to quickly find profiles that you may be interested in following.

profile directory: Ability to view all "accounts".
Explorer: Ability to view popular "posts/hashtags/news/accounts".
explorer = profile directory ? ? ?

**Comment by trwnh** [comment]
the user guide was written for v3.x and needs to be updated for v4.x

in the future please file documentation issues against https://github.com/mastodon/documentation
